tests/pass-dep/libc/libc-mem.rs

```diff
@@ -1,3 +1,4 @@
+//@only-miri: testing specific alignment shenanigans
```

Comment:
Seems unfortunate to skip the *entire* test on the host. Can you instead of `cfg(miri)` to disable just the offending test?